Poker.BE/Poker.BE.Domain/Game/Hand.cs

```diff
@@ -49,37 +56,46 @@ public void DealCards()
             foreach (Player player in activePlayers)
             {
                 for (int i = 0; i < player.PrivateCards.Length; i++)
+                {
                     player.PrivateCards[i] = deck.PullCard();
+                    player.PrivateCards[i].CardState = Card.State.FaceDown;
+                }
             }
         }
 
-        public void PlaceBlinds()
+        public void PlaceBlinds(GamePreferences preferences)
         {
-            PlaceSmallBlind();
-            PlaceBigBlind();
-            PlaceAnts();
+            PlaceSmallBlind(preferences);
```

Comment:
true. preferences should be wrapped with Decorator Design Pattern and contain the policies of the small blind, game preferences. i think that any configurable method should contain preferences argument. like:
PlaceSmallBlind();
PlaceBigBlind();
PlaceAnts();